.github/actions/spelling/expect.txt

```diff
@@ -31,6 +32,7 @@ bomgar
 BOMs
 brk
 Buf
+Bversion
```

Comment:
Add this to the `patterns.txt` file instead:
```
REQUIRE\(RestHelper::GetRestAPIBaseUri\(".*"\) == L".*"

# URL escaped characters
\%[0-9A-F]{2}
```

The second is from https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples:-patterns